Description: correct mysql-global configuration
Author: gisl@leonde.de
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=808729
Forwarded: no
Reviewed-by: Steve Robbins <smr@debian.org>

mysql-global.conf uses an unsupported option 
that causes the db server to not start at all.

See https://bugs.mysql.com/bug.php?id=68315

MySQL 5.1.3 table_cache is depreciated as per change log
(table_open_cache is replaced with table_cache).  In MySQL 5.5, we can
still use table_cache while in MySQL 5.6 mysqld fails to start when
table_cache is used instead of table_open_cache.




--- digikam-4.14.0.orig/core/data/database/mysql-global.conf
+++ digikam-4.14.0/core/data/database/mysql-global.conf
@@ -20,7 +20,7 @@
 #lower_case_table_names=1
 character_set_server=latin1
 collation_server=latin1_general_ci
-table_cache=200
+table_open_cache=200
 thread_cache_size=3
 log_bin=mysql-bin
 expire_logs_days=3
